src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -68,12 +69,14 @@ function BulkEmailForm(props) {
    * @returns formatted Data
    */
   const formatDataForFormAction = (action) => {
+    const emailsFormat = emailLearnersList.map(({ email }) => email);
```

Comment:
What is emailsFormat? Can we change the name to something self-explanatory? Also, I'd prefer using: individualStudentsList